pkg/actors/reminders/scheduler.go

```diff
@@ -124,10 +122,12 @@ func (s *scheduler) CreateReminder(ctx context.Context, reminder *internal.Creat
 		},
 	}
 
-	_, err = s.clients.Next().ScheduleJob(ctx, internalScheduleJobReq,
-		retry.WithMax(3),
-		retry.WithPerRetryTimeout(time.Second/2),
-	)
+	client, err := s.clients.Next(ctx)
+	if err != nil {
+		return fmt.Errorf("error getting scheduler client: %w", err)
+	}
+
+	_, err = client.ScheduleJob(ctx, internalScheduleJobReq)
```

Comment:
do we want the `grpc.WaitForReady(true)` here as well?

Reply:
No it isn’t necessary